docs/DOCUMENTATION.md

```diff
@@ -8,7 +8,9 @@ Overview of the input/output requirements for VECTOR. Initially, there will be a
 
 objectType: string
 
-`sphere, cylinder, plate, plate model`
+`cylinder, sphere, plate, plate model`
+
+![Objects](vector_objects.png)
```

Comment:
@jennyknuth, do you know if this image will show up on your webpage even if it is a relative link? I'm not sure how you're embedding the Markdown and whether this will cause issues on the external website.

Reply:
I am betting that it will show up fine. After the webpage renders, it makes a call to the raw markdown file in GitHub, in this case, https://raw.githubusercontent.com/SWxTREC/vector-code/master/docs/DOCUMENTATION.md, so if the link appears and works there, it should render on the front end…fingers crossed. Worth a try? 

But to be safe, you might want to link directly to the GitHub hosted png source. We could make an 'assets' or 'images' folder and put all images there so their hosted urls will be predictable.

Reply:
Since this image may be superseded by other images/options in the future, I think I'm going to leave it as is and this will also test whether the relative link works or not.